Assets/Scripts/NPC/NpcOrdering.cs

```diff
@@ -18,6 +19,9 @@ public sealed class NpcOrdering : MonoBehaviour
         private int _currentOrder;
         private int _correctDishes;
 
+        [SerializeField] private UnityEvent onStartOrdering = new UnityEvent();
+        [SerializeField] private UnityEvent onFinishOrdering = new UnityEvent();
```

Comment:
```suggestion
        [SerializeField] private UnityEvent onDeliverOrder = new UnityEvent();
        [SerializeField] private UnityEvent onFinshedOrder = new UnityEvent();
```
Dit is het zelfde als start van het order. Je krijgt het. Dan kan je een item van een lijst inleveren. Dan kan het mogelijk zijn dat alles is ingeleverd. Dus naam updaten.